<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenGL - Ray Casting representation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>Eye</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Eye_x_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>-150.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Eye_y_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>150.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Eye_z_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>-150.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>Look_At</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="La_x_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="La_y_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="La_z_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>A View Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Av_x_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Av_y_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Av_z_2">
         <property name="minimum">
          <double>-9990.000000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Luzes">
         <property name="text">
          <string>Luzes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GLWidget" name="GLwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>do_save()</slot>
    <slot>do_load()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
